articles/active-directory-b2c/tutorial-register-applications.md

```diff
@@ -50,8 +50,9 @@ To register a web application in your Azure AD B2C tenant, you can use our new u
 
     The following restrictions apply to redirect URIs:
 
-    * The reply URL must begin with the scheme `https`.
+    * The reply URL must begin with the scheme `https`, unless using localhost.
```

Comment:
```suggestion
    * The reply URL must begin with the scheme `https`, except for localhost redirect URLs.
```

Reply:
I copied the wording from the "Applications (Legacy)" blade thinking it looked better if they were in sync. Specifying "localhost redirect URL" is more specific though, so maybe that should be carried out in both places? (Though I'm guessing the legacy blade will eventually go away.)

Reply:
Oooh, I see. You can commit my suggested change so that I can sign off the PR. Thanks.